pkg/metadata/consts.go

```diff
@@ -0,0 +1,9 @@
+package metadata
+
+const (
+	// AnnotationPrefix is the prefix for Kong annotations.
+	AnnotationPrefix = "konghq.com"
+
+	// TagKey is the key for the tags annotation.
+	TagKey = "/tags"
```

Comment:
Having the `ExtractTags` use them, to minimize the amount of exported symbols we might leave these private. WDYT?

Reply:
👍 f59ee6b2263763bde8235c912f3acca57d6564eb